src/main/kotlin/racingcar/InputValidator.kt

```diff
@@ -0,0 +1,10 @@
+package racingcar
+
+class InputValidator {
+    companion object {
+        fun isNaturalNumber(input: String): Boolean {
+            val number = input.toIntOrNull()
+            return number != null && number > 0
```

Comment:
```suggestion
            return (number != null) && (number > 0)
```

비교에 대한 부분은 그룹을 지어서 표현하는 것이 좋습니다.